digdag-standards/src/main/java/io/digdag/standards/operator/jdbc/CsvWriter.java

```diff
@@ -2,15 +2,16 @@
 
 import java.io.IOException;
 import java.io.Writer;
+import java.io.Closeable;
 import java.util.List;
 import java.util.concurrent.Exchanger;
 
-public class CSVWriter
-    implements AutoCloseable
+public class CsvWriter
+    implements Closeable
```

Comment:
Why did you change the interface to `Closable`? Just for limiting an exception class from `close()` method?